cli-docs/pow/pow_storage-info_list.md

```diff
@@ -0,0 +1,29 @@
+## pow storage-info list
```

Comment:
Nit: Might be good to rename to `query` to be in sympathy with the API? (or rename `Query` to `List`).
I think both are fine, just mentioning they both use different names.

Reply:
Made it `List`